sources/packages/web/src/views/aest/institution/ApplicationOfferingChangeRequestForm.vue

```diff
@@ -185,11 +197,19 @@ export default defineComponent({
         await assessApplicationOfferingChangeRequestModal.value.showModal(
           applicationOfferingChangeRequestStatus,
         );
-      if (responseData) {
+      if (responseData && typeof responseData === "object") {
```

Comment:
what was the reason for the object check here? a boolean will only be sent during cancel, and that too is `false`.  Correct me if I am wrong, 
if the intention was to restrict the type, maybe we can do something like 
![image](https://github.com/bcgov/SIMS/assets/77353155/310ff5e3-5c7c-49db-a449-60bae6e25d36)
 right?